### DISCUSSION TITLE
Output state-space matrix from 'LinearAssembler' solver

### DISCUSSION THREAD

**Discussion by Alanpwu** [ImperialCollegeLondon/sharpy#132] [Q&A] [answered]
Hi, is there a way to configure the 'LinearAssembler' solver such that it outputs the linear state-space matrix (A & B) ? 

Thank you very much!

**Comment by ngoiz** [comment, marked as answer]
Have a look at the `SaveData` post-processor (in `postprocs/savedata.py`, for some reason the docs don't always compile for this postproc). Adding it at the end of the `flow` after you have assembled the linear system will save it to file (make sure you turn the setting `save_linear = on`.

**Reply by Alanpwu** [reply]
it works, thank you!
